Description: Use packaged msgpack
 Remove references to the bundled ./include path from MyBuilder.pm so that the
 headers from the libmsgpack-dev package are used instead.
Author: Jonas Smedegaard <dr@jones.dk>
Reviewed-by: Christopher Hoskin <mans0954@debian.org>
Forwarded: not-needed
Last-Update: 2021-07-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/builder/MyBuilder.pm
+++ b/builder/MyBuilder.pm
@@ -12,7 +12,6 @@
         cc_warnings => 1,
         generate_ppport_h => 'xs-src/ppport.h',
         generate_xshelper_h => 'xs-src/xshelper.h',
-        include_dirs => ['include'],
         needs_compiler_c99 => 1,
         xs_files => { 'xs-src/MessagePack.xs' => 'lib/Data/MessagePack.xs' },
     );
